docs/docs/connect/data-source/azure.md

```diff
@@ -14,12 +14,9 @@ sidebar_position: 05
 azure://<account>.blob.core.windows.net/<container>/path/file.csv
 ```
 
-<img src='/img/connect/data-sources/abs.png' class='rounded-gif' style={{width: '75%', display: 'block', margin: '0 auto'}}/>
-<br />
+## Connect to Azure Blob Storage
 
-## Rill Developer (Local credentials)
-
-When using Rill Developer on your local machine, Rill will use credentials configured in your local environment using the Azure CLI (`az`) or explicitly defined [credentials in a connector YAML](/reference/project-files/connectors#azure).
+When using Rill Developer on your local machine, Rill will use credentials configured in your local environment using the Azure CLI (`az`) or explicitly defined [credentials in a connector YAML](/reference/project-files/connectors#azure) via the Add Data UI.
```

Comment:
We should be conscious of whether we're using system-centric language (e.g. "Rill will use credentials...") or user-centric language (e.g. "You can configure..."), and we should be consistent. In this section, it feels like it should be user-centric (as it is in the "Inferred Credentials" section).